graphql/src/types.ts

```diff
@@ -1,29 +1,29 @@
-import { GraphQlToElasticLanguageMap } from './constants';
+import { type GraphQlToElasticLanguageMap } from './constants';
 
 export type ConnectionCursor = string;
 
-export type ConnectionCursorObject = {
+export interface ConnectionCursorObject {
   offset: number;
-};
+}
 
-export type PageInfo = {
+export interface PageInfo {
   startCursor: ConnectionCursor | null;
   endCursor: ConnectionCursor | null;
   hasPreviousPage: boolean;
   hasNextPage: boolean;
-};
+}
 
-export type ConnectionArguments = {
+export interface ConnectionArguments {
   after?: ConnectionCursor | null;
   first?: number | null;
-};
+}
 
-export type ElasticSearchPagination = {
+export interface ElasticSearchPagination {
```

Comment:
Automatic type→interface conversion have been done here. I don't remember seeing these kind of automatic conversions being done in e.g. events-helsinki-monorepo. Probably stemming from a different configuration?

Reply:
It's because of this eslint rule that comes from the eslint standard that we use: https://typescript-eslint.io/rules/consistent-type-definitions/#when-not-to-use-it

Reply:
...However, this does not mean that I would especially want to use interfaces instead of types. That was actually an accident. I can disable the rule for consistency between the projects..